packages/fast-components-react-base/src/dialog/dialog.props.ts

```diff
@@ -45,6 +45,14 @@ export interface DialogHandledProps extends DialogManagedClasses {
      */
     modal?: boolean;
 
+    /**
+     * Defines where focus goes when the dialog is unmounted.
+     * This is useful because modal dialogs will take focus back until that point in their life cycle.
+     * The element or ref provided will be focused when the dialog unmounts.
+     * Alternatively, providing a callback function gives developers more flexibility if needed.
+     */
+    refocusTarget?: React.RefObject<any> | HTMLElement | (() => void);
```

Comment:
I think we should consider alternative names here so we don’t paint ourselves into a corner semantically. Refocus implies returning focus to a previously focused thing - there are valid scenarios where you may need to move focus to something which previously did not have focus. 

What about something along the lines of ‘focusTargetOnClose’, ‘focusTargetOnUnkount’, etc?

Reply:
renamed to focusTargetOnClose